### ISSUE TITLE
Including ti84pce.h in multiple files causes linktime warnings.

### ISSUE THREAD

**Issue by elfprince13** [CE-Programming/toolchain#16]
If I `#include <ti84pce.h>` in multiple files, at link time, the object files for some of them emit duplicate symbol warnings for `__frameset`:
```
WARNING (909) (00000000 rb_tree\misc.obj)--> Public symbol "__frameset" already externally defined.
WARNING (909) (00D1D3EE rb_tree\misc.obj)--> Public symbol "__frameset" already externally defined.
```

If I rewrite these later files with `#pragma asm` to provide equates for only the labels I need, the warning is not emitted.

**Comment by adriweb** [comment]
Tagging this as an external issue until proven otherwise, as include guars are already present :(

**Comment by debrouxl** [comment]
Include guards protect against multiple inclusion of a given header during a single compiler invocation, but they're powerless against single or multiple inclusion of a given header during multiple independent compiler invocations.
Now, why exactly is the toolchain complaining about __frameset specially, which is an _equate_ in the ASM file included by ti84pce.h, I don't know.

**Comment by adriweb** [comment]
Oh, yeah, "multiple independent compiler invocations", you're probably right...

**Comment by jacobly0** [comment]
WARNING (909) (00000000 Z:\opt\CEdev\lib\std\chelp.lib)--> Public symbol "__imuls" already externally defined.
WARNING (909) (00D1CEC1 Z:\opt\CEdev\lib\std\chelp.lib)--> Public symbol "__imuls" already externally defined.

I believe this is related.

**Comment by elfprince13** [comment]
Perhaps turning the equates into a mix of defines and xrefs would help?

**Comment by elfprince13** [comment]
I think "external issue" is not the right tag for this. It's not a compiler issue, it's a "the include file doesn't behave like an include file, because it introduces link-time symbols, rather than merely providing forward declarations" issue.

**Comment by adriweb** [comment]
Yeah, removed it.

**Comment by MateoConLechuga** [comment]
Just use _At rather than equ.

**Comment by MateoConLechuga** [comment]
The best solution would be to redo the standard libraries to use the libraries in the bootcode. This probably means modifying the crtl source so it doesn't cause these warnings.

**Comment by MateoConLechuga** [comment]
Fixed with release v2.0
